pagefaultgames/pokerogue
# [Qol] Make i18n money formatter controlled by translators

> [!TIP]
> https://github.com/pagefaultgames/pokerogue-locales/pull/6

## What are the changes the user will see?

n/a

## Why am I making these changes?

Right now the money-formatter is hard-coded per language.
This should not be the case but instead be controlled by the translators in the locales directory

## What are the changes from a developer perspective?

The i18n money formatter now calls a new key `common:money` to display the money correctly.

> [!NOTE]
> Updates the locales submodule to https://github.com/pagefaultgames/pokerogue-locales/commit/56eeb809eb5a2de40cfc5bc6128a78bef14deea9

### Screenshots/Videos

see https://github.com/pagefaultgames/pokerogue-locales/pull/6

## How to test the changes?

1. make sure to check out the locales repo branch
2. Start
3. Activate something where you can see the money formatter (e.g. FUn & Games ME)
4. Check the visuals

## Checklist
